<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>도서번호</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>도서명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>작가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>출판사</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>대출 상태</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>00000001</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>박사가 사랑한 수식</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>오가와 요코</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>이레</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>대출 안됨</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>00000002</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>이기적 유전자</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>리처드 도킨스</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>을유문화사</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>대출 안됨</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>00000003</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>박경미의 수학콘서트 플러스</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>박경미</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>동아시아</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>대출 안됨</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>회원 목록</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>회원번호</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>이름</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>이메일</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>연락처</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>대출가능여부</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>대출중인 도서</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>E0000001</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>김철수</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>kim@gmail.com</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>010-1111-1111</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>대출 가능</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>없음</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>E0000002</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>김영희</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>you@naver.com</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>010-2222-2222</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>대출 가능</string>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string>없음</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>E0000003</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>홍길동</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>hong@daum.net</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>010-3333-3333</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>대출 가능</string>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string>없음</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>393</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>대출</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>반납</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>도서 목록</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>최대 대출 가능 권 수 : 1</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>386</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>도서번호</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>도서명</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>작가</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>출판사</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>대출상태</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>501</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>회원번호</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>이름</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>이메일</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>연락처</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>대출가능여부</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>대출중인 도서</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
